strawberryfields/ops.py

```diff
@@ -220,6 +220,23 @@ def apply(self, reg, backend, **kwargs):
         # call the child class specialized _apply method
         return self._apply(temp, backend, **kwargs)
 
+    @staticmethod
+    def _check_for_complex_args(arguments, gate_info):
```

Comment:
This method can now be used by gates for which we'd like to check for complex arguments